### DISCUSSION TITLE
Errors при  открытии  в сдк

### DISCUSSION THREAD

**Discussion by s30-zip** [PavelBlend/blender-xray#439] [Q&A] [answered]
Может у кого было и лечилось, перекинул мэш на кости ТЧ вроде всё проверил, но при открытии в АЕ ошибка, поиски в инете  что-то не помогли

FATAL ERROR
[error]Expression    : assertion failed
[error]Function      : i:\xray-svn\xrcore\FixedVector.h
[error]File          : i:\xray-svn\xrcore\FixedVector.h
[error]Line          : 38
[error]Description   : id<count
[lesnik.zip](https://github.com/PavelBlend/blender-xray/files/7418945/lesnik.zip)

**Comment by PavelBlend** [comment, marked as answer]
@s30-zip у меня нет такой ошибки. Открыл твой blend файл в blender 2.79 и сделал экспорт в object. Модель открылась в AE от SDK 0.4:
![01](https://user-images.githubusercontent.com/7983249/138906818-7208d991-47bb-4c8f-929c-fea520198f3e.jpg)

Можешь скинуть полученный при экспорте object файл, который не открывается в сдк?

**Comment by s30-zip** [comment]
короче, то ли сбой какой в блендере был (хотя перезапускал не однократно), сейчас нормально после экспорта открылось и ogf на object экспортируется. Вот до этого кидал человеку не работал 
[lesnik.zip](https://github.com/PavelBlend/blender-xray/files/7419582/lesnik.zip)

**Reply by PavelBlend** [reply]
@s30-zip в object файле есть скелет, но меш не имеет групп вершин. Я пока не знаю, почему они не сохранились. Посмотрел код и пока не понятно, в каких ситуациях группы не сохранятся. Ты модель не редактировал? В lesnik.blend файле точно получалось сохранить нерабочий object? Возможно была нестандартная ситуация и аддон с ошибкой отработал.

**Comment by s30-zip** [comment]
Я привязал сетку на кости ТЧ и сохранил  .blend, экспортировал в object и кинул куратору мода  ещё днём, вечером прочитал об ошибках, открыл .blend экспортировал вновь, попробовал открыть в АЕ - ошибки, но там ещё  [error]Line : 43 [error]Description : _Pos<size() и [error]Line : 38 [error]Description : 0==dbg_lock были, посмотрел, нет вертекс групп для двух костей думал может в этом причина, создал, хотя они пустые, также не был применён поворот на меше - применил, после этих действий экспортировал несколько раз, object создавался, но ошибка 38 при открытии в АЕ не уходила. Далее вспомнил, что скелет лесника ещё с суставами неправлеными ограничениями (ковертировал старым конвертером, колени в другую сторону сгибаются) думаю поправлю импортирую .ogf новичка, копирую значения ограничений. Пробую опять экспортировать и открыть - фиг, ошибка 38. Думаю может проблема с АЕ ( чтоб проверить будет ли ошибка 38)  импортирую ogf новичка, экспортирую в object , а оно выдаёт ошибку плюс (на скрине я обвёл) создавлась какая-то группа вершин. Потом я сохранил файлы и отписал сюда. Когда ты ответил что всё работает, думаю , может я спятил, проверил экспорт ogf в object - работает, попробовал лесника экспортировать и открыть, тоже работает. Вооот
